contracts/pallet-ibc/src/channel.rs

```diff
@@ -368,6 +369,16 @@ where
 		Ok(())
 	}
 
+	fn store_raw_acknowledgement(
+		&mut self,
```

Comment:
does it need `&mut self`?

Reply:
That's the convention used in ibc-rs for for Keeper traits.

Reply:
If it is not needed, I suggest removing, as mutable references are less flexible in usage in general.